packages/ui-dom-utils/src/getClassList.ts

```diff
@@ -85,8 +109,7 @@ function shimClassListForIE() {
         HTMLElement.prototype,
         'classList'
       )
-      // @ts-expect-error ts-migrate(2345) FIXME: Argument of type 'PropertyDescriptor | undefined' ... Remove this comment to see the full error message
-      Object.defineProperty(SVGElement.prototype, 'classList', descr)
+      descr && Object.defineProperty(SVGElement.prototype, 'classList', descr)
```

Comment:
shimClassListforIE is an old IE workaround, we can remove it

Reply:
Done, thanks